Content.Shared/GameObjects/Components/Utensil/SharedUtensilComponent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Robust.Shared.GameObjects;
+
+namespace Content.Shared.GameObjects.Components.Utensil
+{
+    [Flags]
+    public enum UtensilKind : byte
```

Comment:
I think UtensilType sounds way better. Kind sounds "kind"a weird

Reply:
Changed it, originally named it after SuicideKind